client/src/pages/ThesisPage/components/ThesisCommentsList/ThesisCommentsList.tsx

```diff
@@ -0,0 +1,91 @@
+import { useThesisCommentsContext } from '../../../../contexts/ThesisCommentsProvider/hooks'
+import {
+  Button,
+  Card,
+  Center,
+  Group,
+  Modal,
+  Pagination,
+  Skeleton,
+  Stack,
+  Text,
+} from '@mantine/core'
+import { IThesisComment } from '../../../../requests/responses/thesis'
+import { useState } from 'react'
+import AuthenticatedFilePreview from '../../../../components/AuthenticatedFilePreview/AuthenticatedFilePreview'
+import { useLoggedInUser } from '../../../../hooks/authentication'
+import { formatDate, formatUser } from '../../../../utils/format'
+import { Download } from 'phosphor-react'
+
+const ThesisCommentsList = () => {
+  const { thesis, comments, deleteComment, limit, page, setPage } = useThesisCommentsContext()
+
+  const user = useLoggedInUser()
+
+  const [openedComment, setOpenedComment] = useState<IThesisComment>()
+
+  return (
+    <Stack>
+      <Modal opened={!!openedComment} onClose={() => setOpenedComment(undefined)} size='xl'>
+        {openedComment && (
+          <AuthenticatedFilePreview
+            url={`/v2/theses/${thesis.thesisId}/comments/${openedComment.commentId}/file`}
+            filename={`comment-${openedComment.commentId}`}
+            height={400}
+          />
+        )}
+      </Modal>
+      {!comments &&
+        Array.from(Array(limit).keys()).map((index) => <Skeleton key={index} height={50} />)}
+      {comments?.content.map((comment) => (
```

Comment:
When there are no comments to display, consider adding a fallback message indicating that there are no comments yet. This provides better feedback to the user instead of just showing nothing.

Reply:
I will add that